spec/models/partners/profile_spec.rb

```diff
@@ -265,6 +271,22 @@
         profile.reload
         expect(profile.client_share_total).to eq(99)
         expect(profile.valid?).to eq(false)
+        expect(profile.errors[:base]).to include("Total client share must be 0 or 100")
+      end
+
+      it "sets error at field level when feature flag enabled for partner step form" do
+        allow(Flipper).to receive(:enabled?).with("partner_step_form").and_return(true)
```

Comment:
Can you add the `and_return(false)` to the previous method? It's always best to call out explicitly when you depend on a value to be true/false rather than relying on defaults.

Reply:
Done.